src/api/subscriptions.js

```diff
@@ -337,6 +337,13 @@ export class Subscriptions {
    * @return {?}
    */
   showBestAudienceAction() {}
+
+  /**
+   * Set the publisherProvidedId.
+   * @param {string} publisherProvidedId
+   * @return {?}
+   */
+  setPpid(publisherProvidedId) {}
```

Comment:
Lets use the full name in our public interfaces `setPublisherProvidedId`

Reply:
okay will update